### PR TITLE
#3049 - Autotests for Toolbar ui

### PR DESCRIPTION
**Pull request by krakmielepam**
## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

## Check list
- [ ] unit-tests written
- [x] e2e-tests written
- [x] documentation updated
- [x] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [x] task status changed to "Code review"

### CODE REVIEW

File: ketcher-autotests/tests/User-Interface/Toolbar/toolbar-ui.spec.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      */
+    await takeLeftToolbarScreenshot(page);
+  });
+
+  test('Help: UI Verification', async ({ page }) => {
+    /*
+        Test case: Test case: EPMLSOPKET-1328
+        Description: Help button tooltip verification
+        */
+    await page.getByTitle('Help (?)').first().hover();
```
Comment (Nitvex) [inline ketcher-autotests/tests/User-Interface/Toolbar/toolbar-ui.spec.ts:36]: JFYI, "tooltip" is shown by browser, not by Ketcher. And the tooltip itself just depends on "title" attribute of DOM-node. https://www.w3schools.com/tags/att_global_title.asp#:~:text=HTML%20title%20Attribute&text=The%20title%20attribute%20specifies%20extra,validate%20on%20any%20HTML%20element.

Actually, you can simplify the test (and future tests) just by (example code, may not work):
```
const helpButton = page.getByTestId('help-button');
await expect(someButton).toHaveAttribute('title', 'Help (?)');
```

You can refer: https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-attribute

File: ketcher-autotests/tests/User-Interface/Toolbar/toolbar-ui.spec.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        Test case: Test case: EPMLSOPKET-1330
+        Description: Menu bar buttons verification
+        */
+    await takeTopToolbarScreenshot(page);
+
+    await pressButton(page, STRUCTURE_LIBRARY_BUTTON_NAME);
+    await page.getByRole('tab', { name: 'Functional Groups' }).click();
+    await selectFunctionalGroups(FunctionalGroups.Bn, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await takeTopToolbarScreenshot(page);
```
Comment (Nitvex) [inline ketcher-autotests/tests/User-Interface/Toolbar/toolbar-ui.spec.ts:53]: Could you clarify, why do we have screenshots of top toolbar multiple times even though it doesn't change?

File: ketcher-autotests/tests/User-Interface/Toolbar/toolbar-ui.spec.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    await resetCurrentTool(page);
+
+    await selectLeftPanelButton(LeftPanelButton.RectangleSelection, page);
+    await page.getByRole('button', { name: 'Lasso Selection (Esc)' }).click();
```
[the remark this answers is not in the document]
Comment (krakmielepam) [reply ketcher-autotests/tests/User-Interface/Toolbar/toolbar-ui.spec.ts:57]: oki

File: ketcher-autotests/tests/User-Interface/Toolbar/toolbar-ui.spec.ts (hunk 2 above)
Comment (krakmielepam) [reply ketcher-autotests/tests/User-Interface/Toolbar/toolbar-ui.spec.ts:53]: they differ: https://jiraeu.epam.com/browse/EPMLSOPKET-1330 
some of the buttons are not enabled

**Pull request merged**